src/Compilers/Test/Core/Assert/ConditionalFactAttribute.cs

```diff
@@ -159,6 +159,7 @@ public static class ExecutionConditionUtil
         public static bool IsWindowsDesktop => IsWindows && IsDesktop;
         public static bool IsMonoDesktop => Type.GetType("Mono.Runtime") != null;
         public static bool IsMono => MonoHelpers.IsRunningOnMono();
```

Comment:
Should we remove this property or change the logic to be `IsMonoDesktop || IsMonoCore`?

Reply:
(Filing an issue here is fine as well.)